crates/db_schema/src/source/post.rs

```diff
@@ -165,14 +168,16 @@ pub struct PostLike {
   pub published: DateTime<Utc>,
 }
 
-#[derive(Clone)]
+#[derive(Clone, derive_new::new)]
 #[cfg_attr(feature = "full", derive(Insertable, AsChangeset))]
 #[cfg_attr(feature = "full", diesel(table_name = post_actions))]
 pub struct PostLikeForm {
   pub post_id: PostId,
   pub person_id: PersonId,
   #[cfg_attr(feature = "full", diesel(column_name = like_score))]
   pub score: i16,
+  #[new(value = "naive_now()")]
```

Comment:
This `naive_now()` function is really useless, would be shorter and clearer to remove it and use `Utc::now()` directly.

Reply:
Sounds good. I could probably remove that function as part of this PR anyway.